recipe/meta.yaml

```diff
@@ -22,28 +22,32 @@ requirements:
     - pip
     - python
     - setuptools
+    - wheel
   run:
     - python
     - pytest >=3.6.0
```

Comment:
pytest should be >=5.0.0, see https://github.com/pytest-dev/pytest-timeout/blob/8e4800ecbcf149790bb8035ec37dc30f3d48a127/setup.py#L19